doc/changes/DM-48032.bugfix.rst

```diff
@@ -0,0 +1 @@
+Include non-Python data artifacts when packaging for PyPi distribution, including `etc/*yaml` and `final_post.sh`
```

Comment:
```suggestion
Include non-Python data artifacts when packaging for PyPi distribution, including ``etc/*yaml`` and ``final_post.sh``
```
Since this is rst and not markdown.